pages/uuid.tsx

```diff
@@ -42,3 +46,18 @@ export default function UuidPage() {
     </Layout>
   );
 }
+
+const i18nextNameSpaces: Namespace[] = ['common', 'uuid'];
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => {
```

Comment:
Can we use the [provider](https://react.i18next.com/latest/i18nextprovider) component instead of putting `getStaticProps` on every page?

Reply:
https://github.com/i18next/next-i18next#3-project-setup

I have tried and it seems necessary.
`appWithTranslation` is used in _app.tsx. I think this `appWithTranslation` is what corresponds to provider.

Reply:
I see, you are correct. Does the `useTranslation` hook work here instead of using `getStaticProps`?

Reply:
If you feel `getStaticProps` is best here, we can stick with that.

Reply:
I think it is best to use `getStaticProps `.
It is also quite natural for the getStaticProps method to be present when adding pages in Next.js.

Reply:
That works for me, thanks!